app/pages/project/components/ProjectNavbar/components/ProjectTitle/ProjectTitle.jsx

```diff
@@ -25,7 +25,31 @@ const StyledLink = styled(IndexLink).attrs({
   }
 `;
 
-function ProjectTitle({ link, title }) {
+function ProjectTitle({ launched, link, title, underReview }) {
+  if (underReview) {
```

Comment:
The duplication doesn't seem necessary here. The check for the states can be done inline:
```
<H1>
  <StyledLink to={`${link}?facelift=true`}>
    {underReview &&
      <p><em>Under Review</em></p>
    {title}
    {launched &&
      <img />}
  </StyledLink>
</H1>
```

Reply:
Good point thanks.